app/src/main/java/com/g11x/checklistapp/NavigationActivity.java

```diff
@@ -66,6 +67,15 @@
   @Override
   protected void onCreate(Bundle savedInstanceState) {
     super.onCreate(savedInstanceState);
+    overridePendingTransition(android.R.anim.fade_in, android.R.anim.fade_out);
+    PreferredLanguageSupport.applyPreferredLanguage(this);
+    final Activity thisActivity = NavigationActivity.this;
```

Comment:
I'm transitioning the variable reference from the local scope to effectively member scope in an anon class so it needs to be final in Java. Many times you'll see parameters in methods declared final for this reason as well.